style.css

```diff
@@ -414,3 +429,293 @@ form {
   align-items: center;
   padding-bottom: 32px;
 }
+
+@media only screen and (min-width: 768px) {
+  /* For Desktop */
+  body {
+    margin: 0;
+    padding: 0;
+    background-image: none;
+    overflow: hidden;
+    overflow-y: scroll;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .navbar ul {
+    padding-top: 24px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 40%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #1a2236;
+  }
+
+  .navbar ul li a:hover {
+    text-decoration: underline;
+    text-underline-offset: 8px;
+  }
+
+  .hello {
+    text-decoration: underline;
+    text-underline-offset: 8px;
+  }
+
+  .hero {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    margin-top: 16px;
+    justify-content: center;
+  }
+
+  .hero-1 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 48px;
+    width: 100%;
+    text-align: center;
+    padding-left: 25%;
+    box-sizing: border-box;
+    position: relative;
+  }
+
+  .heading {
+    font-family: 'Playfair Display', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 110px;
+    line-height: 100%;
+    text-align: left;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .para {
+    text-align: left;
+  }
+
+  .btn-a {
+    display: flex;
+    justify-content: start;
+  }
+
+  .btn {
+    width: 45%;
+  }
```

Comment:
- [ ] Kindly consider implementing the button interaction style as it is shown on the Figma template when the user `hover, press and disable`
Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199659552-63d1b3f5-299d-4f03-8edc-3305a6e7cca8.png)